content/applications/finance/payment_providers/demo.rst

```diff
@@ -0,0 +1,129 @@
+=====================
+Demo payment provider
+=====================
```

Comment:
```suggestion
====
Demo
====
```

"payment provider" is in the category. We wouldn't repeat "payment provider" for all pages in this category. "demo" is no exception.